Docker-compose/docker-compose.yml

```diff
@@ -1,12 +1,11 @@
+name: iam-proxy-italia
 services:
-
-  satosa-mongo:
+  mongo:
```

Comment:
there might be different containers about mongo, and this is the mongo within the satosa network with a specialized name

Reply:
The container name is composed by project name +service name + istance number = iam-proxy-italia-mongo-1
The image name is composed by project name +service name = iam-proxy-italia-mongo
But you can call it with the service name in composer

Reply:
The project name is defined in [first docker-compose line](https://github.com/italia/iam-proxy-italia/blob/4666dd0ce980cf19611f34e9c5584829d4646cca/Docker-compose/docker-compose.yml#L1) with key `name: iam-proxy-italia`